CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+### Consistent sampling
```

Comment:
@SylvainJuge I believe you wised to call out some breaking changes. If so please provide the language to be included in the change log.

Reply:
thanks, I've added #2033